syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -505,6 +665,16 @@ def relu(tensor_shares):
 
                 module.relu = relu
 
+                def pad(input, pad, mode="constant", value=0):
+                    padded_shares = {}
+                    shares_dict = input.child.child if input.is_wrapper else input.child
+                    for location, shares in shares_dict.items():
+                        padded_shares[location] = torch.nn.functional.pad(shares, pad, mode, value)
+
+                    return AdditiveSharingTensor(padded_shares)
+
+                module.pad = pad
+
```

Comment:
You should use the decorator `@overloaded.function` which helps you to have a simpler code:
```
                @overloaded.function
                def pad(input_shares, pad, mode="constant", value=0):
                    padded_shares = {}
                    for location, shares in input_shares.items():
                        padded_shares[location] = torch.nn.functional.pad(shares, pad, mode, value)

                    return padded_shares

                module.pad = pad
```